erpnext/crm/doctype/lead/lead.py

```diff
@@ -24,21 +23,36 @@ class Lead(SellingController, CRMNote):
 	from typing import TYPE_CHECKING
 
 	if TYPE_CHECKING:
-		from frappe.types import DF
-
 		from erpnext.crm.doctype.crm_note.crm_note import CRMNote
+		from erpnext.crm.doctype.lead_channel.lead_channel import LeadChannel
+		from erpnext.crm.doctype.lead_diamond.lead_diamond import LeadDiamond
+		from erpnext.crm.doctype.lead_product_item.lead_product_item import LeadProductItem
+		from frappe.types import DF
 
+		account_number: DF.Data | None
+		address: DF.Data | None
 		annual_revenue: DF.Currency
+		bank_branch: DF.Literal[None]
+		bank_district: DF.Literal[None]
+		bank_name: DF.Literal["Agribank", "ACB", "BIDV", "Vietcombank", "VietinBank", "Techcombank", "Sacombank", "MB Bank", "Eximbank", "VPBank", "SHB", "NamABank", "B\u1eafc \u00c1 Bank", "OceanBank", "TPBank"]
```

Comment:
non-blocking: Why are the others using international name, but Bắc Á?

![image](https://github.com/user-attachments/assets/7d8e95c6-a6e1-4fd0-b31e-648b78e4f60c)